File: shoprest/serializers/product.py
from rest_framework import serializers

from shopengine.models import Product

class ProductSerializer(serializers.ModelSerializer):
	class Meta:
		model = Product
		fields = (
			'id',
			'name',
			'slug',
			'meta_description',
			'short_description',
            'description',
            #'image',
            #'thumbnail',
            'author',
            'unit_price',
            'is_active',
            'product_type',
            #'parent',
		)

File: shopie/assets/tmp/core_object-cache_path-RKnKcjkF.tmp/shopie/models/product.js
define('shopie/models/product', ['exports', 'ember-data'], function (exports, DS) {

    'use strict';

    exports['default'] = DS['default'].Model.extend({
        name: DS['default'].attr('string'),
        slug: DS['default'].attr('string'),
        metaDescription: DS['default'].attr('string'),
        shortDescription: DS['default'].attr('string'),
        description: DS['default'].attr('string'),
        image: DS['default'].attr('string'),
        thumbnail: DS['default'].attr('string'),
        unitPrice: DS['default'].attr('string'),
        isActive: DS['default'].attr('boolean'),
        productType: DS['default'].attr(),
        parent: DS['default'].belongsTo('product', { inverse: 'variants' }),
        variants: DS['default'].hasMany('product', { inverse: 'parent' }),
        author: DS['default'].belongsTo('user', { async: true }),

        formatSlug: (function () {
            return this.get('slug') + '-' + this.get('id');
        }).property('slug', 'id')
    });

});